### ISSUE TITLE
test stk_param_estim.m on octave 8.2.1: multiple FAILures 

### ISSUE THREAD

**Issue by dasergatskov** [stk-kriging/stk#39]
```
Failure Summary:

  ..tave/api-v58/packages/stk-2.8.0/param/estim/stk_param_estim.m  pass    6/11  
                                                                   FAIL    5
```
They all look similar, e.g.:
```
***** test  % noiseless
 zi = stk_feval (f, xi);
 param1 = stk_param_estim (model, xi, zi, param0);
 assert (isequal (size (param1), size (param0)))
warning: sqp: QP subproblem is non-convex and unbounded
warning: called from
    sqp at line 436 column 9
    stk_minimize_boxconstrained at line 49 column 14
    stk_param_estim_optim at line 70 column 23
    stk_param_estim at line 139 column 15
    __test__ at line 4 column 9
    test at line 682 column 11

!!!!! test failed
operator *: nonconformant arguments (op1 is 3x6, op2 is 0x0)
shared variables     f =

@(x) (-(0.8 * x + sin (5 * x + 1) + 0.1 * sin (10 * x)))

    xi =

      <class stk_factorialdesign>

    zi = [](0x0)
    NI = 20
    param0 =

            0
       1.3863
       0.9163

    param1 = [](0x0)
    model =

      scalar structure containing the fields:

        covariance_type = @stk_materncov_iso
        lm =

          <class stk_lm_constant>

        dim = 1
        param =

           NaN
           NaN
           NaN

        lognoisevariance = -Inf

```

**Comment by dasergatskov** [comment]
It looks like there is a regression in sqp function in octave 8.2+:
https://savannah.gnu.org/bugs/?64302

**Comment by jbect** [comment]
Thank you @dasergatskov :+1:  I will look into it.

**Comment by jbect** [comment]
Closed by commit ecf6be8d4a94a6ddd28cebcff44308287ce419c8
